### DISCUSSION TITLE
Allow object browser search to include the member text

### DISCUSSION THREAD

**Discussion by Sdthedawson** [codefori/vscode-ibmi#1215] [Ideas]
Just another idea i think that would be a great addition.
![image](https://user-images.githubusercontent.com/1506998/229919336-d2fe3999-4337-46ff-9583-8939469b917a.png)
  
Right now the object browser search works perfectly on the file names. But since the member text is also shown it would be neat if the search could also work on it.

**Comment by WBSpmichal** [comment]
Just throwing a +1 for this. 

At my company, every single program was named using an esoteric naming scheme, meaning none of them reflect their actual purpose (unless you memorize all the codes and such).

Right now I can only use search to find a program if I already know what it's called. If search also included the description / member text that would fix it for me. 

![image](https://github.com/user-attachments/assets/c48d66d2-c407-494b-b96f-c83281e2908e)

**Comment by sebjulliand** [comment]
Unfortunatelly this search feature here
![image](https://github.com/user-attachments/assets/0dc0376f-31ec-41ed-a902-39a97e56e76d)
is a standard search widget provided by VS Code on every tree view. We have no control over its behavior.

If we want to be able to run a search for members based on their text, we'll need a specific action like this one:
![image](https://github.com/user-attachments/assets/0462033a-5e8c-441c-a3b2-78aa060b16be)

**Comment by WBSpmichal** [comment]
when generating the list of members, could you just combine the name and text so vscode sees it as a single string? 

The search source member does work pretty well also, thanks for letting me know about that.
